OpenSim/Simulation/Test/testModelInterface.cpp

```diff
@@ -35,14 +35,6 @@ int main() {
 
     try {
         Model model("arm26.osim");
-        // finalizeFromProperties() is required to build internal ownership tree
-        // attempt to access the ComponentList will throw that the model (root)
-        // has no subcomponents
-        ASSERT_THROW(ComponentIsRootWithNoSubcomponents, 
-            model.countNumComponents());
-
-        // finalize internal data structures from its properties
-        model.finalizeFromProperties();
 
         // all subcomponents are now accounted for.
```

Comment:
Consider adding a note that this is because `finalizeFromProperties()` is called in the constructor.